.github/workflows/test-verification.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           echo "PYTHONUNBUFFERED=$PYTHONUNBUFFERED" >> $GITHUB_ENV
           TEST_PATH=$RV_ROOT/verification/top/${{ matrix.test }}
           echo "TEST_PATH=$TEST_PATH" >> $GITHUB_ENV
-          echo "HTML_FILE=${FULL_NAME}.html" >> $GITHUB_ENV
+          echo "HTML_FILE=${FULL_NAME}_${{ matrix.COVERAGE }}.html" >> $GITHUB_ENV
```

Comment:
nitpick: could be `matrix.coverage` for consistency but it seems it doesn't matter